migrations/1664273420286_add-table-to-record-likes-on-posts.js

```diff
@@ -0,0 +1,24 @@
+
+
+exports.shorthands = undefined
+
+exports.up = (pgm) => {
+	pgm.createTable('post_likes', {
+		id: 'id',
```

Comment:
we won't be using an `id` here, 

as the issue says, we want to use the `userId` and `postId` as a composite primary key. Kindly do  take a look into that

Reply:
yes please

Reply:
my understanding of the composite primary key means both fields will have a primarykey set to true right?

Reply:
not sure if i get what you mean but it means both fields combined will form the primary key of the table

Reply:
so I will give some sort of direction here since might be a first

This is a bit of insight into it: https://linuxhint.com/postgres-composite-primary-key/

After reading we can look into how to translate it using the migration library we're using from their docs: https://salsita.github.io/node-pg-migrate/#/tables

Kindly reach out if anything is not clear

Reply:
yes please, i have an idea of how to go about it

Reply:
i have applied the change now , kindly review

Reply:
Seen, thanks

 I must admit I was sceptical about how it looked thinking it would not work but the library seems to handle it just fine. 

Notwithstanding, I would want us to let it look like the normal SQL kind where the composite primary key is formed after the field definitions. Something like this should achieve the same effect

```js
exports.up = (pgm) => {
	pgm.createTable('post_likes', {
		userId: {
			type: 'integer',
			notNull: true,
			references: '"users"',
			onDelete: 'cascade'
		},
		postId: {
			type: 'integer',
			notNull: true,
			references: '"posts"',
			onDelete: 'cascade'
		},
		createdAt: 'createdAt'
	}, {
		constraints: {
			primaryKey: ['userId', 'postId']
		}
	})
}

```

Reply:
alright, thanks a lot

Reply:
i have pushed the changes